l10n_fi_edicode/data/res_partner_operator_einvoice.xml

```diff
@@ -0,0 +1,212 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
```

Comment:
1. Yes, we can drop `<data/>` in 13.
2. It isn't look like someone actually update these data even during migration from one version to another. So, I think it better to have `noupdate` (it wasn't here before) to leave it up on users to update this information.

But let me know what do you think, I will do as you decide

Reply:
1. Then you can have `<odoo noupdate="1">` instead, right? And drop the `data` tags.
2. This is a philosophical question. If there's no `noupdate`, then we indeed can update the existing records, but it also mean that we *must* do so and that brings greater responsibility and requires that someone is actually monitoring the changes in the operators. Otherwise we'll have stale data and might accidentially update an already updated record with old information.

One kind of hacky solution would be to drop the `noupdate` and make the `res.partner.operator.einvoice` set its own XML ID to `noupdate` mode if someone modifies the record by hand. That way only the untouched records would get updated from the XML and the rest would be left intact.

Reply:
done